packages/starksheet-webapp/src/react-app-env.d.ts

```diff
@@ -1 +1,4 @@
 /// <reference types="react-scripts" />
+interface Window {
```

Comment:
required for accessing the window.ethereum provider and providing type support. the `ethereum: any` is ugly but I'm not aware of a better way of typing it (using a matching type from a lib does not work either)